Formula/voltdb.rb

```diff
@@ -14,13 +14,21 @@ class Voltdb < Formula
   end
 
   depends_on :ant => :build
+  depends_on "cmake" => :build
 
   def install
     system "ant"
 
-    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader"],
+    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader", "bin/voltdeploy", "bin/voltenv"],
       %r{VOLTDB_LIB=\$VOLTDB_HOME\/lib}, "VOLTDB_LIB=$VOLTDB_HOME/lib/voltdb"
 
+    inreplace "bin/voltenv" do |s|
+      s.gsub! /VOLTDB_VOLTDB="\$VOLTDB_LIB"/, "VOLTDB_VOLTDB=\"$VOLTDB_BASE/voltdb\""
+
+      # Remove voltenv installed as link check
+      s.gsub! /if \[ "\$\{0\}" = "\$SOURCE" \]; then/, "if [ \"${0}\" = \"${BASH_SOURCE[0]}\" ]; then"
```

Comment:
Should definitely speak to upstream about this check to ensure there's not other things that will break here.